README.md

```diff
@@ -1,49 +1,60 @@
-[![Build Status](https://travis-ci.org/RocketRobz/TWiLightMenu.svg?branch=master)](https://travis-ci.org/RocketRobz/TWiLightMenu)
-# ![TWiLightMenu++](https://github.com/RocketRobz/TWiLightMenu/blob/master/logo.png)
+<p align="center">
+ <img src="https://github.com/RocketRobz/TWiLightMenu/blob/master/logo.png"><br>
```

Comment:
Reason why I didn't want to do this is because the bg isn't transparent. It just doesn't look good